extension-base/src/main/java/com/azure/autorest/extension/base/model/codemodel/Operation.java

```diff
@@ -82,9 +82,9 @@ public class Operation extends Metadata {
     private ConvenienceApi convenienceApi;
 
     /**
-     * a boolean to represent should generate protocol api or not
+     * a boolean to represent should generate protocol api or not, default is true
      */
-    private boolean generateProtocolApi;
+    private boolean generateProtocolApi = true;
```

Comment:
For consistency, use `Boolean`.

This is code model.
Whether null means true/false the logic of Mapper.